workbench-core/environments/src/index.ts

```diff
@@ -12,21 +12,31 @@ import {
 import { SortAttribute, isSortAttribute } from './constants/sortAttributes';
 import EnvironmentTypeHandler from './handlers/environmentTypeHandler';
 import StatusHandler from './handlers/statusHandler';
+import {
+  CreateEnvironmentTypeConfigRequest,
+  CreateEnvironmentTypeConfigRequestParser
+} from './interfaces/createEnvironmentTypeConfigRequest';
```

Comment:
This is not an interface, I'd move it to `models` or better `schemas` / `parsers`

Reply:
I've seen other projects use `models` folder, so to follow convention I will change the name of folder to `models`

Reply:
changed folder to models to follow convention